src/docs/ArcCommanderManual/arc_isa_investigation.md

```diff
@@ -84,8 +87,8 @@ add sidebar: _sidebars/mainSidebar.md
 | AuthorList                 | The list of authors associated with that publication.                                                                                                                                       | John P Doe; Eva Muller  |
 | Title                      | The title of publication associated with the investigation.                                                                                                                                 |                         |
 | Status                     | A term describing the status of that publication (i.e. submitted, in preparation, published).                                                                                               | in preparation          |
-| StatusTermAccessionNumber  | The accession number from the Term Source associated with the selected term.                                                                                                                |                         |
-| StatusTermSourceREF        | Identifies the controlled vocabulary or ontology that this term comes from. The Source REF has to match one the Term Source Name declared in the in the Ontology Source Reference section.  |                         |
+| StatusTermAccessionNumber  | The accession number from the Term Source associated with the selected term.                                                                                                                |   http://www.ebi.ac.uk/efo/EFO_0001795                      |
```

Comment:
I guess this is supposed as an example?

Reply:
Yes, for the example "in preparation" for the status above.